packages/storybook-aem/app/cmds/init/questions/packageJSON.js

```diff
@@ -2,20 +2,22 @@ const fs = require('fs');
 const path = require('path');
 const prompts = require('prompts');
 const cwd = process.cwd();
+const { exec } = require('child-process-promise');
 
 module.exports = async (args, config) => {
     let fileLocation = false;
     let filename = false;
 
-    try { 
-        if (fs.existsSync(path.resolve(cwd, config.projectRoot, config.relativeProjectRoot, 'package.json'))) {
-            fileLocation = 'root';
-            filename = `./package.json`;
-        } else if (fs.existsSync(path.resolve(cwd, config.projectRoot, config.relativeProjectRoot, config.uiApps, 'package.json'))) {
-            fileLocation = 'ui.apps';
-            filename = `${config.uiApps}/package.json`;
-        }
-    } catch(err) { throw err; }
+    if (fs.existsSync(path.resolve(cwd, config.projectRoot, config.relativeProjectRoot, 'package.json'))) {
+        fileLocation = 'root';
+        filename = `./package.json`;
+    } else if (fs.existsSync(path.resolve(cwd, config.projectRoot, config.relativeProjectRoot, config.uiApps, 'package.json'))) {
+        fileLocation = 'ui.apps';
+        filename = `${config.uiApps}/package.json`;
+    } else if (fs.existsSync(path.resolve(cwd, 'package.json'))) {
+        fileLocation = 'cwd';
+        filename = '.' + path.resolve(cwd, 'package.json').replace(path.resolve(cwd, config.projectRoot, config.relativeProjectRoot), '');
+    }
```

Comment:
Here I added the present working directory to one of the places that gets checked for a package.json